src/main/java/com/teragrep/k8s_01/config/AppConfigKubernetes.java

```diff
@@ -57,4 +58,14 @@ public int getCacheMaxEntries() {
     public String toString() {
         return new Gson().toJson(this);
     }
+
+    @Override
+    public void validate() throws InvalidConfigurationException {
```

Comment:
this does not validate if labels are missing completely or any other json subobject